ui/src/app/components/apps-panel/expanded-app.tsx

```diff
@@ -160,7 +168,7 @@ const RStudioButtonRow = (props: {
   const { userApp, workspaceNamespace } = props;
   const [launching, setLaunching] = useState(false);
```

Comment:
since we've changed the meaning of "launching", these should now be creating/setCreating

Reply:
Done